tests/models/test_taskinstance.py

```diff
@@ -869,7 +869,7 @@ def test_xcom_pull(self):
         # Pull the values pushed by both tasks
         result = ti1.xcom_pull(
             task_ids=['test_xcom_1', 'test_xcom_2'], key='foo')
-        self.assertEqual(result, ('bar', 'baz'))
+        self.assertEqual(result, ['baz', 'bar'])
```

Comment:
Thanks @ashb for reviewing. Appreciate it.

I've updated this test because it was failing before. The `xcom_pull` would invoke the `XCom.get_one()` many times, and transform it into an iterable. Therefore ordering would be lost. Now we get the xcom's descending by execution_date, insertion_date.